packages/pieces/math/src/lib/actions/division.ts

```diff
@@ -0,0 +1,25 @@
+import { createAction, PieceAuth, Property, Validators } from "@activepieces/pieces-framework";
+
+export const division = createAction({
+    name: 'division_math',
+    auth: PieceAuth.None(),
+    displayName: "Division",
+    description: 'Divide first number by the second number',
+    props: {
+        first_number: Property.Number({
+            displayName: 'First Number',
+            description: undefined,
+            required: true,
+            validators: [Validators.number]
+        }),
+        second_number: Property.Number({
+            displayName:  'Second Number',
+            description: undefined,
+            required: true,
+            validators: [Validators.number, Validators.nonZero]
```

Comment:
You can use Validator.minValue(1) instead and then you can remove Validators.NonZero

Reply:
What if we want to generate a number between -5 and 5?

Reply:
@lisander-lopez This is division action, I think we dont need non zero for generate random numbee

Reply:
Oh sorry, I was reading the wrong code. The reason I put non-zero is because you cannot divide by 0. but you can divide by a negative number